crawler/cv/spiders/huxiu.py

```diff
@@ -1,13 +1,50 @@
 # -*- coding: utf-8 -*-
 import scrapy
+from cv.items.article import ArticleItem
 
 
 class HuxiuSpider(scrapy.Spider):
     name = "huxiu"
     allowed_domains = ["huxiu.com"]
     start_urls = (
-        'http://www.huxiu.com/',
+        #'http://www.huxiu.com/',
+        'http://www.huxiu.com/article/146942',
     )
+    custom_settings = {
+        'ITEM_PIPELINES': {
+            'cv.pipelines.a36kr.ArticlePipeline': 300
+        }
+    }
 
     def parse(self, response):
-        pass
+        print 'start'.center(100, '=')
+        item = self.parse_page(response)
+        print item['author_link']
+        print 'end'.center(100, '=')
+        yield item
+
+    def parse_page(self, response):
+        sel = response.selector
+        item = ArticleItem()
+        item['url'] = response.url
+        item['title'] = sel.xpath('//title/text()').extract()[0]
+        item['content'] = sel.css('#article_content').extract()[0]
+        item['summary'] = None
+        item['published_ts'] = None
+        item['created_ts'] = None
```

Comment:
We need created_ts and updated_ts are current utc time. Please just keep them like 36kr crawler.